dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -131,16 +131,18 @@ const merchandisingAdContainerStyles = css`
 
 const merchandisingAdStyles = css`
 	position: relative;
-	min-height: ${adSizes.billboard.height + labelHeight}px;
-	margin: 12px auto;
+	min-height: ${adSizes.billboard.height + labelHeight + space[3]}px;
+	margin: ${space[3]}px auto;
 	max-width: ${breakpoints['wide']}px;
 	overflow: hidden;
+	padding-bottom: ${space[3]}px;
```

Comment:
What's the rationale for using `min-height`, `margin` and `padding` for the spacing?

Reply:
Okay makes sense! Why is the `+ space[3]` to the min-height needed? I see there used to be + 20 below for desktop, not sure why that was/is either 🤔